src/background.js

```diff
@@ -0,0 +1,34 @@
+BROWSER_SDK.storage.sync.get("domains", function(items) {
+  var patterns = null;
```

Comment:
There was a subtle race condition in the old code where the browser storage callback didn't fire fast enough before the context menus were created. It's important for the `contextMenu.create` calls to happen within the callback from `browser.storage.sync.get`.

Reply:
Thanks, for this one in particular. I have always blamed Firefox for the menu not appearing